src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -178,6 +178,21 @@ static constexpr Property<std::string, PropertyMutability::RO> model_name{"NETWO
 static constexpr Property<uint32_t, PropertyMutability::RO> optimal_number_of_infer_requests{
     "OPTIMAL_NUMBER_OF_INFER_REQUESTS"};
 
+namespace element {
+inline std::istream& operator>>(std::istream& is, element::Type& type) {
```

Comment:
@ilya-lavrenov Not completely sure if we really need this
But without it element::Type is not "Istreamable" and cannot be converted to string
I will cover all the types if we decide to keep it here.

Reply:
The same logic is needed for gna_plugin
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gna/gna_plugin_config.cpp#L43

Reply:
looks like the comment is not valid anymore..